esque/cli/commands.py

```diff
@@ -311,10 +311,21 @@ def transfer(
             return
 
         state.config.context_switch(to_context)
-        _produce_from_file(topic, to_context, working_dir, avro)
+        _produce_from_files(topic, to_context, working_dir, avro)
 
 
-def _produce_from_file(topic: str, to_context: str, working_dir: pathlib.Path, avro: bool):
+@esque.command("produce", help="Produce messages from directory based on output from transfer command")
+@click.argument("topic", required=True)
+@click.option(
+    "-d", "--directory", help="Sets the directory that contains Kafka messages", type=click.STRING, required=True
```

Comment:
I would prefer to hand in a path instead of a directory. You could use the pathlib's globbing feature to hand in folders by passing `folder/*.avro`. Still would give us the possibility to only hand in a file.

Reply:
We will do this in https://github.com/real-digital/esque/issues/70